tests/acceptance/porter/test_decentralized_porter.py

```diff
@@ -111,8 +76,7 @@ def test_exec_work_order(blockchain_porter,
     # Setup
     network_middleware = MockRestMiddleware()
     # enact new random policy since idle_blockchain_policy/enacted_blockchain_policy already modified in previous tests
-    enacted_policy = random_blockchain_policy.enact(network_middleware=network_middleware,
-                                                    publish_treasure_map=False)  # enact but don't publish
+    enacted_policy = random_blockchain_policy.enact(network_middleware=network_middleware)  # enact but don't publish
```

Comment:
I guess the "but don't publish" comment is redundant now

Reply:
✔️